tests/schema_test.py

```diff
@@ -0,0 +1,51 @@
+from aepp.schema import Schema
+import unittest
+from unittest.mock import patch, MagicMock
+
+
+class SchemaTest(unittest.TestCase):
```

Comment:
It would be useful to go even further in these tests, where we mock not just the function call, but the actual functionality - for exaple for `test_schema_get_tenant_id` can we mock the actual return value from the API and use that to validate that the tenant ID value we get is indeed the expected one?

For example see https://realpython.com/testing-third-party-apis-with-mocks/#mocking-the-complete-service-behavior for how to do this.

Reply:
yes thanks for the great feedback, will dig into next level of detail

Reply:
It would be great to test that as I, and I assume you as well, will probably build business logic out of the default response from the AEP API.